packages/fxa-auth-server/test/scripts/bulk-mailer.js

```diff
@@ -211,15 +211,17 @@ describe('scripts/bulk-mailer', function () {
         const test2Html = fs
           .readFileSync(path.join(OUTPUT_DIRECTORY, 'user2@test.com.html'))
           .toString();
-        assert.include(test2Html, 'Este es un correo automatizado');
+        assert.include(test2Html, 'Confirma tu cuenta');
```

Comment:
I had to change this since we don't currently have "This is an automated email" translated in `es` `auth.ftl` yet so I picked another `es` string.